code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -15,9 +15,11 @@
 	container_type = OPENCONTAINER
 	slot_flags = SLOT_FLAG_BELT
 	/// If TRUE, the hypospray can inject through most hardsuits/modsuits.
-	var/can_pierce_hardsuits = FALSE
+	var/penetrate_thick = FALSE
```

Comment:
does this include biosuits?

Reply:
Yes, it pierces all thick clothing. The only clothes that block it are those that have hypospray immunity (unless the hypo already ignores that), I'll update the comment to reflect this.